src/mlpack/methods/det/CMakeLists.txt

```diff
@@ -4,11 +4,7 @@
 set(SOURCES
   # the DET class
   dtree.hpp
-  dtree.cpp
-
-  # the util file
-  dt_utils.hpp
-  dt_utils.cpp
+  dtree_impl.hpp
```

Comment:
I've decided to remove them, because I'm not sure they should actually be part of the library itself. Aren't they mostly related to the command-line tool?